war/src/main/java/io/lumeer/engine/controller/DatabaseInitializer.java

```diff
@@ -69,6 +69,14 @@ public void init() {
                .append(LumeerConst.UserRoles.ATTR_PROJECT_ID, LumeerConst.Index.ASCENDING)
                .append(LumeerConst.UserRoles.ATTR_USER_ROLE, LumeerConst.Index.ASCENDING), true);
       }
+
+      if (!dataStorage.hasCollection(LumeerConst.Organization.COLLECTION_NAME)) {
+         dataStorage.createCollection(LumeerConst.Organization.COLLECTION_NAME);
+         dataStorage.createIndex(LumeerConst.Organization.COLLECTION_NAME, new DataDocument(LumeerConst.Organization.ATTR_ORG_ID, LumeerConst.Index.ASCENDING), true);
+         dataStorage.createIndex(LumeerConst.Organization.COLLECTION_NAME, new DataDocument(LumeerConst.Organization.ATTR_ORG_NAME, LumeerConst.Index.ASCENDING), true);
```

Comment:
According to the discussion on the meeting, the last parameter in this method should be `false` so there will not be unique index on organization names.